### ISSUE TITLE
single gpu mode edge loss undefined

### ISSUE THREAD

**Issue by twangnh** [lxasqjc/Deformation-Segmentation#6]
hi @lxasqjc as I could not run into the multi-GPU mode data parallel, I was trying to do a step-by-step running with single GPU mode, but I notice that the edge loss is undefined if single gpu mode is used:

```
        if single_gpu_mode:
            loss, acc, _ = segmentation_module(batch_data[0], writer=writer, count=cur_iter, epoch=epoch)
        else:
            if cfg.TRAIN.opt_deform_LabelEdge and epoch >= cfg.TRAIN.fix_seg_start_epoch and epoch <= cfg.TRAIN.fix_seg_end_epoch:
                loss, acc, edge_loss = segmentation_module(batch_data)
            elif cfg.TRAIN.deform_joint_loss:
                loss, acc, edge_loss = segmentation_module(batch_data)
            else:
                loss, acc = segmentation_module(batch_data)
```

it raises edge_loss undefined later, could you please help? is it ok to just modify single gpu mode as :
```
        if single_gpu_mode:
            loss, acc, edge_loss = segmentation_module(batch_data[0], writer=writer, count=cur_iter, epoch=epoch)

```

**Comment by lxasqjc** [comment]
Thanks for point out and yes you are right, single_gpu_mode is now fixed, see [6d11f56](https://github.com/lxasqjc/Deformation-Segmentation/commit/6d11f56f4ee9fd02579dd15538dd5c90ca214bd0)
